README.md

```diff
@@ -267,6 +267,15 @@ Default: empty
 Specify a comma-separated list of IPv4 CIDRs to exclude from SNAT. For every item in the list an `iptables` rule and off\-VPC
 IP rule will be applied. If an item is not a valid ipv4 range it will be skipped. This should be used when `AWS_VPC_K8S_CNI_EXTERNALSNAT=false`.
 
+#### `POD_MTU` (v1.x.x+)
+
+Type: Integer as a String
+
+*Note*: The default value is set to AWS_VPC_ENI_MTU, which defaults to 9001 if unset.
+Default: 9001
+
+Used to configure the MTU size for pod virtual interfaces. The valid range is from `576` to `9001`.
```

Comment:
`576` is only valid for IPv6, otherwise the minimum MTU has to be `1280`

Reply:
Good call @archoversight. @jchen6585 we should update the README for `AWS_VPC_ENI_MTU` and `POD_MTU` at the very least, and we can consider adding runtime logging to error if IPv6 is configured and the MTU is < 1280